src/main/java/org/jabref/gui/preferences/entry/EntryTab.fxml

```diff
@@ -54,4 +54,15 @@
     <Label styleClass="sectionHeader" text="%Time stamp"/>
     <CheckBox fx:id="addCreationDate" text="%Add timestamp to new entries (field &quot;creationdate&quot;)"/>
     <CheckBox fx:id="addModificationDate" text="%Add timestamp to modified entries (field &quot;modificationdate&quot;)"/>
+
+
+
+    <CheckBox fx:id="enableViewTracking" text="%Enable user behaviour analysis"/>
```

Comment:
`Enable user behaviour analysis` --> wrong word - please try to use the same words "view tracking" below "popularity" is OK, but "bheavior analysis" is even more general. It should be consistent with src/main/java/org/jabref/gui/groups/GroupDialog.fxml.